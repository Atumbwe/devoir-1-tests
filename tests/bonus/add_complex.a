2 2 5 6 complex +