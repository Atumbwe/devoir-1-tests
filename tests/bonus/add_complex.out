 ______
< 7+8i >
 ------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
